src/it/java/io/deephaven/benchmark/tests/standard/parquet/ParquetCodecTest.java

```diff
@@ -0,0 +1,95 @@
+package io.deephaven.benchmark.tests.standard.parquet;
+
+import org.junit.jupiter.api.*;
+import org.junit.jupiter.api.MethodOrderer.OrderAnnotation;
+
+/**
+ * Standard tests for writing/reading multi-column data with different codec/compression. To save time, the parquet
+ * generated by the "write" tests is used by the "read" tests
+ */
+
+// Need to add array and vector tests. Look at adding iterations
+
+@TestMethodOrder(OrderAnnotation.class)
+public class ParquetCodecTest {
+    final ParquetTestRunner runner = new ParquetTestRunner(this);
+    final String[] usedColumns = {"str10K", "long10K", "int10K", "short10K", "bigDec10K", "array1K", "vector1K"};
```

Comment:
I added the ones Ryan asked for.  Typically, we don't try to go for full test coverage for each operation, because the nightly runs would take too long.  (They already take too long.)

Reply:
Got it, thanks for explaining. You can mark this resolved.